app/main/lib/similarity.py

```diff
@@ -119,3 +136,12 @@ def get_similar_items(item, similarity_type):
   app.logger.info(f"[Alegre Similarity] [Item {item}, Similarity type: {similarity_type}] response for search was {response}")
   return response
 
+def validate_item(item, similarity_type):
```

Comment:
I don't see this function called anywhere in the diff. Where do we use it? Are `assert` statements sufficient or do we need to log?